app/views/notes/show.html.erb

```diff
@@ -78,6 +78,17 @@
   <div style="overflow:hidden;" id="content">
   <%= raw auto_link(@node.latest.render_body, :sanitize => false) %>
   </div>
+  
```

Comment:
You can make a new folder for location-related partials, too -- like `/app/views/location/`, if you think that's worthwhile?